src/psyclone/psyir/nodes/if_block.py

```diff
@@ -60,8 +60,14 @@ class IfBlock(Statement):
     # with the Fortran 'case' or C 'switch' syntactic constructs;
     # 'was_where' - a conditional structure originally implied by a Fortran
     # WHERE construct.
+    # 'was_select_type' to tag a conditional structure originally
```

Comment:
Please update the Dev Guide (https://psyclone-dev.readthedocs.io/en/latest/psyir.html#control-flow-node-annotation) with this information.

Reply:
Updated. and renamed annotations.